waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -793,7 +793,10 @@ proc diff*(e1, e2: Epoch): int64 =
   let
     epoch1 = fromEpoch(e1)
     epoch2 = fromEpoch(e2)
-  return int64(epoch1) - int64(epoch2)
+  # Cannot use `abs` here because it is not available for `uint64`
+  if epoch1 > epoch2: return int64(epoch1 - epoch2)
+  else: return int64(epoch2 - epoch1) * -1
```

Comment:
```suggestion
  if epoch1 > epoch2: 
    return int64(epoch1 - epoch2)
  else: 	
    return int64(epoch2 - epoch1) * -1
```

Reply:
Does it apply to the `safeIntCast` proc in [730e352](https://github.com/status-im/nwaku/pull/1318/commits/730e352b07edd2cea9e856f595965482b025025c) as well?